h2o-algos/src/test/java/hex/grid/SequentialWalkerTest.java

```diff
@@ -74,6 +72,27 @@ public void test_SequentialWalker() {
             Scope.exit();
         }
     }
+
+    @Test
+    public void test_SequentialWalker_getHyperParams() {
```

Comment:
does the resume work also with parallel grid search?

Reply:
modified the test to test this as well, actually discovered a bug, in parallel grid search resume code